### ISSUE TITLE
Test Snapshots have invalid HTML

### ISSUE THREAD

**Issue by zbruno** [PocketDerm/radiance-ui#38]
SNG components are getting two opening brackets rendered in snapshots.

```js
<<svg viewBox="0 0 10 10" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

**Comment by youngmanalive** [comment]
@snags88 - I think we're good to close this now.

**Comment by snags88** [comment]
agreed 👍

**Comment by snags88** [comment]
closed by 5032878
